app/models/conversation.rb

```diff
@@ -110,6 +110,12 @@ def last_message_less_than_24_hrs?
     Time.current < last_incoming_message.created_at + 24.hours
   end
 
+  def last_message_less_than_7_days?
+    return false if last_incoming_message.nil?
```

Comment:
let's consider whether the human tag is enabled and if so let's compare the 24 hours or else let's do 7 days.